substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/ResourcesFeature.java

```diff
@@ -72,12 +72,16 @@
 public final class ResourcesFeature implements Feature {
 
     public static class Options {
-        @Option(help = "Regexp to match names of resources to be included in the image.", type = OptionType.User)//
+        @Option(help = "Regexp to match names of resources to be included in the image.", type = OptionType.User)
```

Comment:
Please re-add the `//` at the end. Otherwise `mx eclipseformat` wants to put the annotation in the same line as the field.

Reply:
Re-added //